crates/fluvio-cli/src/connector/update.rs

```diff
@@ -0,0 +1,87 @@
+//!
+//! # Update a Managed Connector
+//!
+//! CLI tree to generate Update a Managed Connector
+//!
+
+use fluvio::metadata::topic::ReplicaSpec;
+use structopt::StructOpt;
+use tracing::debug;
+
+use fluvio::{Fluvio, FluvioError};
+use fluvio::metadata::{
+    topic::{TopicSpec, TopicReplicaParam},
+    connector::ManagedConnectorSpec,
+};
+
+use fluvio_sc_schema::ApiError;
+use fluvio_sc_schema::errors::ErrorCode;
+
+use crate::CliError;
+use super::ConnectorConfig;
+
+// -----------------------------------
+// CLI Options
+// -----------------------------------
+
+#[derive(Debug, StructOpt, Default)]
+pub struct UpdateManagedConnectorOpt {
+    /// The configuration file for the update Managed Connector
+    #[structopt(short = "c", long = "config", value_name = "config", required(true))]
+    pub config: String,
+}
+
+impl UpdateManagedConnectorOpt {
+    pub async fn process(self, fluvio: &Fluvio) -> Result<(), CliError> {
+        let config = ConnectorConfig::from_file(&self.config)?;
+        let spec: ManagedConnectorSpec = config.clone().into();
+        let name = spec.name.clone();
+
+        debug!(connector_name = %name, connector_spec = >spec "updating managed_connector");
```

Comment:
I think that I had a typo here, it should be
```suggestion
        debug!(connector_name = %name, connector_spec = ?spec "updating managed_connector");
```